samplesheets/tests/test_views_api.py

```diff
@@ -625,6 +634,160 @@ def test_get_json(self):
         self.assertEqual(response.data, expected)
 
 
+class TestIrodsAccessTicketListAPIView(IrodsAccessTicketAPIViewTestBase):
+    """Tests for IrodsAccessListAPIView"""
+
+    def setUp(self):
+        super().setUp()
+        self.path = self.coll.path
+        self.label = TICKET_LABEL
+        self.url = reverse(
+            'samplesheets:api_irods_ticket_list',
+            kwargs={'project': self.project.sodar_uuid},
+        )
+
+    def test_get(self):
+        """Test IrodsAccessTicketListAPIView GET"""
+        self.ticket = self.make_irods_ticket(
```

Comment:
Minor note: If you create the ticket within a test case instead of `setUp()` or a helper, and nothing is done to it in `tearDown()` or a cleanup method,  you don't need to declare it as a member variable. Simply setting it up as `ticket = ' is better. There is no functional difference but it will save us some redundant characters :)